docs/tutorials/getting_started/mac_osx.rst

```diff
@@ -0,0 +1,74 @@
+Mac/OSX Installation Guide
+==========================
+
+Quickstart
+----------
+The following commands should be sufficient to get you started with installing and using IDAES. For more information on these steps, see below.
+
+.. sidebar:: Learn more about the Installation
+   :class: quickstart-toc
+
+   * `Install Prerequisites`_
+   * `Install IDAES-PSE`_
+   * `Install IDAES Extensions`_
+   * `Install IDAES Examples`_
+   * `Run IDAES Tests`_
+
+.. Import quick start guide, including OS specific lines & skipping non-OS lines & comments
+.. literalinclude:: install_templates/quickstart.md
+   :language: bash
+   :linenos:
+   :lines: 1-5,7,10-
+
+------------------------------------------------
+
+Installing IDAES
+----------------
+To get IDAES fully set up on your machine, we'll go through the steps to get idaes-pse package installed as well as setting up the IDAES extensions, which includes some extra solvers and function libraries, the IDAES example files, and the IDAES tests.
+
+Install Prerequisites
+^^^^^^^^^^^^^^^^^^^^^
+
+**Install  Miniconda**
+
+1. Download `Anaconda <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_
```

Comment:
This says "Anaconda" but the title and link are to Miniconda.  Maybe just change this to "Miniconda" in the href?  And perhaps add a "(or Anaconda, you prefer)" after the title, before the link?

This is the same on the Windows and Linux pages as well...